authorizer.js

```diff
@@ -1,14 +1,14 @@
 const { isAuthorized, runIfDev } = require('./utils')
 
-exports.handler = async function (event) {
+exports.handler = async function(event) {
   const secret = process.env.JWT_SECRET
   if (!secret) {
     console.log('Error acquiring secret from env')
-    throw 'JWTSecretError'
+    throw new Error('JWTSecretError')
   }
 
   if (!isAuthorized(event.authorizationToken, secret)) {
-    throw 'Unauthorized'
+    throw new Error('Unauthorized')
```

Comment:
Throwing the string `Unauthorized` was important to make API Gateway return a 401 error instead of a 403, which the apps rely on. Making this change will require supporting infra changes to mutate the response code.

Reply:
I can instead have eslint ignore this and put in a comment as to why.